casts/rows/src/UIForeground/Parts/SecondaryPart.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { TouchableOpacity, StyleSheet } from 'react-native';
+import { PartStatusContext } from '../Container';
+import type { SecondaryPartProps, PartStatus } from '../types';
+
+export function SecondaryPart({ children, onPress, disabled, negative }: SecondaryPartProps) {
+    const partStatus: PartStatus = React.useMemo((): PartStatus => {
+        return {
+            disabled,
+            negative,
+            partType: 'Secondary',
+            partState: onPress || disabled !== undefined ? 'Pressable' : 'NonPressable',
+        };
+    }, [disabled, negative, onPress]);
+
+    return (
+        <TouchableOpacity
```

Comment:
So it have unnecessary `TouchableOpacity` (with all it's complications) even if `onPress` won't be ever provided?